spotify-api/src/main/java/kaaes/spotify/webapi/android/models/Playlist.java

```diff
@@ -18,15 +18,18 @@ public int describeContents() {
 
     @Override
     public void writeToParcel(Parcel dest, int flags) {
+        super.writeToParcel(dest, flags);
         dest.writeString(this.description);
         dest.writeParcelable(this.followers, 0);
         dest.writeParcelable(this.tracks, 0);
     }
 
     public Playlist() {
+        super();
```

Comment:
This is not really needed but I think we can merge this anyway. I will clean this up with the next commit.